projects/packages/masterbar/src/nudges/additional-css/class-wpcom-additional-css-manager.php

```diff
@@ -54,6 +56,12 @@ public function register_nudge( \WP_Customize_Manager $wp_customize_manager ) {
 	 * @return string
 	 */
 	private function get_nudge_url() {
-		return '/checkout/' . $this->domain . '/premium';
+		return '/checkout/' . $this->domain . '/' . $this->get_plan()->path_slug;
```

Comment:
Should we point to /plans instead? 🤔

Reply:
Why do you think we should point to `/plans`?

Reply:
Is this a scope change? Lets just do the simplest possible thing we can right now, while just swapping premium for personal. We can do follow-ups / consult with design if there are bits of the ux we can improve outside of this project.

Reply:
@rcrdortiz For these reasons:
- For Global Styles, we are pointing them to /plans. It would be consistent with that upsell.
- pointing to the plans will advertise all the benefits of upgrading, not just the CSS
- We'll remove the plan hardcode and promote the `style customization` one
- I think in /plans, for atomic sites, we only advertise Business and eCommerce to free sites - meaning we won't need this second PHP class for Atomic sites

@dsas yup, it's a scope change. That's why I haven't updated it :D

Reply:
Some GS upgrades point to checkout though. It's not unified.